pkg/prompb/labels.go

```diff
@@ -0,0 +1,32 @@
+package prompb
+
+import (
+	"errors"
+	"github.com/prometheus/prometheus/pkg/labels"
+)
+
+// FromLabelMatchers parses protobuf label matchers to Prometheus label matchers.
+func FromLabelMatchers(matchers []*LabelMatcher) ([]*labels.Matcher, error) {
```

Comment:
Nit would prefer this in another package. This package is only for protobuf pseudo-generated code.